orchestrator/domain/base.py

```diff
@@ -893,6 +904,17 @@ def db_model(self) -> SubscriptionInstanceTable:
     def parents(self) -> List[SubscriptionInstanceTable]:
         return self._db_model.parents
 
+    @property
+    def parent_ids(self) -> List[Optional[Union[UUID, UUIDstr]]]:
+        p_ids: List[Optional[Union[UUID, UUIDstr]]] = []
+        if len(self.parents) > 0:
+            p_ids.extend(
+                self.parents.col[idx].parent_id  # type: ignore
+                for idx, ob in enumerate(self.parents.col)  # type: ignore
```

Comment:
can't you directly loop through the self.parents?

Reply:
Tried that before but got DetachedSessionError exception. We only need the id and we do not have to do an explicit fetch of the parent that triggers the DetachedSessionError in some cases

Reply:
ohh oke